error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1204,12 +1204,15 @@ StepVerifier.FirstStep<? extends T> after(Flux<T> flux) {
     }
   }
 
-  /** Don't unnecessarily call {@link StepVerifier.Step#expectNext(Object[])}. */
-  static final class StepVerifierStepExpectNextEmpty<T> {
+  /**
+   * Don't unnecessarily call {@link StepVerifier.Step#expectNext(Object[])} or {@code
+   * StepVerifier.Step#expectNextCount(0L)}.
+   */
```

Comment:
Since we'll most likely extend this in the future:
```suggestion
  /** Don't unnecessarily have {@link StepVerifier.Step} expect no elements. */
```